submissions/rafiatAminu/HTML/html-forms/contact.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
```

Comment:
You have to delete this file. You already have one created inside `html-forms` folder